### PR TITLE
Fix the cors error

### PR DESCRIPTION
**Pull request by MikeLing**
We use flask-cros to enable CROS in flask. You could read it[1] for some more detail.

[1] http://flask-cors.readthedocs.io/en/latest/

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/ouija/173)

### CODE REVIEW

**Comment by MikeLing** [comment]
@jmaher r?

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ Flask-SQLAlchemy==2.1
 MarkupSafe==0.23
 MySQL-python==1.2.5
 psycopg2==2.6.2
+Flask-SQLAlchemy==2.1
+flask-cors>=3.0.2
+argparse==1.2.1
```
Comment (jmaher) [inline requirements.txt]: is argparse needed for cors?  same for Flask-SQLAlchemy?  They seem to be used for other things.

**Comment by MikeLing** [comment]
@jmaher r？

**Comment by jmaher** [comment]
thanks for fixing this!

**Pull request merged**